pkg/oplabels/labels.go

```diff
@@ -43,3 +44,12 @@ func HasManagedByCassandraOperatorLabel(m map[string]string) bool {
 	v, ok := m[ManagedByLabel]
 	return ok && v == ManagedByLabelValue
 }
+
+// CleanLabelName a valid label must be an empty string or consist of alphanumeric characters,
+// '-', '_' or '.', and must start and end with an alphanumeric.
+// Note: we apply a prefix of "cassandra-" to the cluster name value used as label name.
+// As such, empty string isn't a valid case.
+func CleanLabelName(labelName string) string {
+	var regexpResult = whitelistRegex.FindAllString(strings.Replace(labelName, " ", "", -1), -1)
```

Comment:
Just a note, the `var` keyword is not necessary when you can use assignment operator (`:=`). So `regexpResult := whitelist ..` is just as good (and more goish). Only the global-scope ones need the var ones since they're not inside a function.